data/nmstate/002-nmstated.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: nm-state-controller
+  namespace: nmstate
+spec:
+  template:
+    metadata:
+      labels:
+        nmstate.io: state-controller
+    spec:
+      serviceAccountName: nmstate-state-controller
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule
+      containers:
+        - name: state-controller
+          image: {{ .NMStateHandlerImage }}
+          imagePullPolicy: {{ .ImagePullPolicy }}
+          args: ["-execution-type", "client"]
```

Comment:
I think we have a bug in kubernetes-nmstate manifests. This type should be controller, not client. I'm sorry for dragging this PR for so long, but would you please fix it here?